src/plugins/crypto.c

```diff
@@ -970,7 +970,7 @@ gboolean bd_crypto_tc_open (const gchar *device, const gchar *name, const guint8
     gint ret = 0;
     guint64 progress_id = 0;
     gchar *msg = NULL;
-    struct crypt_params_tcrypt params = {0};
+    struct crypt_params_tcrypt params = {NULL,0,NULL,0,NULL,NULL,NULL,0,0,0};
```

Comment:
I will say this right away here: *I don't like this change.* The compiler should be clever enough to figure this out. Maybe *clang* needs to be told to not produce warnings about this?

Reply:
I completely agree with you on this, and there should be a better way to do that. It's possible upstream may have thought of this already and provided a compiler flag or macro for this. I'll investigate.

Reply:
> I'll investigate.

Thanks!